style.css

```diff
@@ -183,6 +183,15 @@ html {
   border: 2px solid var(--outline);
 }
 
+.btn-link{
+  text-decoration: none;
+  color: #fff;
+}
+.btn-link:hover{
+  text-decoration: none;
+  color: #000;
+}
+
```

Comment:
I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

Reply:
> I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

Add it yourself and make another PR

Reply:
> I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

I just reviewed the project and I must admit, I love it. Nice job you did there man 💙

Reply:
> I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

@sandilya27 please when you make the changes let me know

Reply:
@pythoniclaver changes are done please review it and accept the pull request

Reply:
@sandilya27 Our Repository deserve a star from you, don't we?😭😭😭

Drop a star so we would be motivated to  create another issue 🙏.